src/app/backend/client/manager.go

```diff
@@ -97,6 +100,38 @@ func (self *clientManager) Config(req *restful.Request) (*rest.Config, error) {
 	return cfg, nil
 }
 
+// ClientCmdConfig creates ClientCmd Config based on authentication information extracted from request.
+// Currently request header is only checked for existence of 'Authentication: BearerToken'
+func (self *clientManager) ClientCmdConfig(req *restful.Request) (clientcmd.ClientConfig, error) {
```

Comment:
That will cause that in ```Config``` method, we create ```*rest.Config``` at first, then translate it to ```clientcmd.ClientConfig``` and translate it back to ```*rest.Config```, I thought it would increase some complexity. comments?